bower.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
Windows XP is also still working :D
It would have been nice to finally remove the compiled JS.
Btw, it is possible to use `bower install` from a tarball, so you can directly install npm packages from unpkg with Bower if you so desire.
But your decision :)